__tests__/reducers/literal.test.js

```diff
@@ -1,88 +1,136 @@
-import { literal } from '../../src/reducers/literal'
+import { removeAllContent, removeMyItem, setMyItems  } from '../../src/reducers/literal'
 
 describe('literal reducer', () => {
-  it('should handle initial state', () => {
-    expect(
-      literal(undefined, {})
-    ).toEqual({formData: []})
-  })
 
   it('should handle SET_ITEMS', () => {
-    expect(
-      literal({formData: []}, {
-        type: 'SET_ITEMS',
-        payload: {id: 1, uri:'Run the tests', items: []}
-      })
-    ).toEqual({
-      "formData": [{
-        "id": 1, "uri": "Run the tests", "items": []
-      }]
+    const literalSetItems = setMyItems({ "resourceTemplate:Monograph:Instance": {
+      'http://schema.org/name': { items: [] }
+    }}, {
+      type: 'SET_ITEMS',
+      payload: {
+        rtId: 'resourceTemplate:Monograph:Instance',
+        uri:'http://schema.org/name',
+        items: [ { id: 0, content: 'Run the tests'} ]
+      }
+    })
+    expect(literalSetItems).toEqual({
+      "resourceTemplate:Monograph:Instance": {
+        'http://schema.org/name': {
+          items: [{ id: 0, content: 'Run the tests'}]
+        }
+      }
     })
 
     expect(
```

Comment:
this expect seems to warrant its own description -- perhaps "SET_ITEMS appends new items to the existing array of items" 

which makes me wonder ... what happens if you pass in one that is already there?